src/pages/HomePage.js

```diff
@@ -1,8 +1,39 @@
-
+import { useState, useEffect } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import axios from 'axios';
 
 function HomePage() {
-    return(
-        <div>HomePage</div>
+    const navigate = useNavigate();
+
+    const [accessToken, setAccessToken] = useState(localStorage.getItem('access_token') || sessionStorage.getItem('access_token'));
+    const [refreshToken, setRefreshToken] = useState(localStorage.getItem('refresh_token') || sessionStorage.getItem('refresh_token'));
+
+
+    useEffect(() => {
+        if (!accessToken) {
+            navigate('/')
+        }
+    }, [accessToken]);
+
+    const handleLogout = () => {
+        setAccessToken("");
+        setRefreshToken("");
+        localStorage.clear();
+        sessionStorage.clear();
+    };
+
+    return (
+        // TEMPORARY HTML USED FOR TESTING
+        <div className="container">
+            <div style={{width:'450px',wordWrap: 'break-word'}}>
+                Access Token: {accessToken}
```

Comment:
I think this is good to have for even for the project part 1, but maybe instead we can display the author instead of the access token (though not sure if that will be implemented yet by then)

Reply:
I mostly just have this here so that we can see how the token is retrieved and used.